python/format/src/datasets_test.py

```diff
@@ -0,0 +1,52 @@
+from etils import epath
+from format.src import datasets
+from format.src import errors
+import pytest
+
+
+# End-to-end tests on real data. The data is in `tests/end-to-end/*.json`.
+@pytest.mark.parametrize(
+    ["filename", "error"],
+    [
+        # Metadata.
+        [
+            "metadata_missing_property_name.json",
+            'Property "https://schema.org/name" is mandatory, but does not exist.',
```

Comment:
I think the test should check the context too, or there is a chance that one uses mlfield_missing_property_name.json as an input and get a passing test.